libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/components/IconButton.kt

```diff
@@ -36,11 +42,15 @@ fun IconButton(
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
     content: @Composable () -> Unit
 ) {
+    val colors = IconButtonDefaults.iconButtonColors(
+        contentColor = LocalContentColor.current,
```

Comment:
Why are we using `LocalContentColor` here and `ElementTheme.colors` for disabled?

Reply:
The way it was designed in [Figma](https://www.figma.com/file/G1xy0HDZKJf5TCRFmKb5d5/Compound-Android-Components?type=design&node-id=1182%3A48861&mode=design&t=bMycIvAOH66Apkuv-1), the icon button could use either primary or secondary colors, so it made more sense to just allow the parent component to decide which color should be used for the icon.

As for disabled, it's always the same color for icons, so we can just hardcode it.

Reply:
Ok makes sense!